shard.go

```diff
@@ -234,7 +234,9 @@ func (s *Shard) close() error {
 	if s.store != nil {
 		_ = s.store.Close()
 	}
-	s.stats.Inc("close")
```

Comment:
The other problem it would appear is something is calling code when the server is shut down, which doesn't seem right either.